module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -169,6 +169,51 @@ Read the full documentation and download cheatsheets (where available) for these
 
 ### `Seurat` and `SCE` object conversion
 
+This [vignette](https://satijalab.org/seurat/articles/conversion_vignette.html) from the `Seurat` documentation explains how to convert between `SCE` and `Seurat` formats.
+
+In addition, we provide some code examples below for how you can accomplish these conversions.
+For all code examples below, it is assumed that `SingleCellExperiment` and `Seurat` libraries have been loaded in your R environment:
+
+```r
+library(Seurat)
+library(SingleCellExperiment)
+```
```

Comment:
I think we may not want to do this, or we may want to only half do this. I think it is important to know which of the functions we are using is from `Seurat` vs. SCE, so I might load SCE, but leave `Seurat` functions prefixed with `Seurat::`

Reply:
Done in https://github.com/AlexsLemonade/training-modules/pull/668/commits/8ce618227f1993746208c2cd57611dc20f700112